lib/xclarity_client/xclarity_base.rb

```diff
@@ -16,7 +16,11 @@ def initialize(conf, uri)
     def connection_builder(conf, uri)
       $lxca_log.info "XClarityClient::XClarityBase connection_builder", "Creating connection to #{conf.host + uri}" 
       #Building configuration
-      @conn = Faraday.new(url: conf.host + uri) do |faraday|
+      host = conf.host;
+      if !host.start_with?('https://') && !host.start_with?('http://')
+         host = 'https://' + host
```

Comment:
When I told about the port, it's other concern that we should analyze, maybe more problematic than if always will run over HTTPS. But we can discuss it in another moment.

Once the LXCA will always run over HTTPS protocol, as you said, it LGTM! 😄 👍